workers/data_refinery_workers/processors/illumina.py

```diff
@@ -157,11 +157,14 @@ def _detect_columns(job_context: Dict) -> Dict:
                 # Since there are multiple annotations, we need to break early before continuing.
                 # Related: https://github.com/AlexsLemonade/refinebio/issues/499
                 continue_me = False
-                for annotation in sample.sampleannotation_set.all():
-                    if annotation.data.get('description', '') == header:
-                        column_ids = column_ids + str(offset) + ","
-                        continue_me = True
-                        break
+                for annotation in sample.sampleannotation_set.filter(is_ccdl=False):
+                    try:
+                        if annotation.data.get('description', '')[0] == header:
```

Comment:
You've set yourself up for a string index out of range error if `annotation.data` doesn't contain the `description` key.

Reply:
Maybe that's why you put it in a try block though...

Reply:
Yeah. I don't know why GEO returns the description as 1-length array but it does.